src/navigation.vue

```diff
@@ -45,6 +45,10 @@ Weekdays from 8:30 AM to 4 PM" href="mailto:servicedesk@csc.fi?subject=Fairdata%
 							<font-awesome-icon icon="circle-notch" spin />
 						</b-nav-text>
 
+						<b-nav-item v-else-if="$auth.getLoginError()" key="invalid-user-logout" @click="logout()">
```

Comment:
should this be as well computed value?

Reply:
Just tested. It won't work here as a computed value because it's not a reactive value, so it would be read only when the navigation component is mounted, i.e. when the app starts. The reason this works here is a side effect of things like routing changes causing a re-rendering of the navigation component, which is not pretty to rely on. Also $auth.loggedIn has the same issue. The public interface of the $auth plugin could use a redesign to make everything reactive.

Reply:
ok